osschallenge/tests/pages/register.py

```diff
@@ -0,0 +1,27 @@
+class RegisterPage(object):
+
+    def __init__(self, selenium_driver, live_server_url):
+        self.driver = selenium_driver
+        self.live_server_url = live_server_url
+
+    def open(self):
+        self.driver.get("{}{}".format(self.live_server_url, "/register/"))
+        return self
+
+    def register(
+            self, user, first_name, last_name, email, password1, password2
```

Comment:
nitpick: Do you really need to pass the same password two times?

Reply:
As discussed I don't change the code here.